app/src/main/java/com/douglasbruce/tasky/core/data/repository/ReminderRepositoryImpl.kt

```diff
@@ -32,15 +33,17 @@ class ReminderRepositoryImpl @Inject constructor(
     }
 
     override suspend fun createReminder(reminder: AgendaItem.Reminder): AuthResult<Unit> {
-        //TODO: Create local
+        dao.upsertReminder(reminder.toReminderEntity())
+
         return authenticatedRetrofitCall(serializer) {
             taskyNetwork.createReminder(reminder.toCreateReminderRequest())
             AuthResult.Success(Unit)
         }
     }
```

Comment:
I assume this is all WIP, but here you'd also need to take care of what happens when the API call fails, since in that case the previously inserted item would need to be synced with the backend later